src/lib/modules/blockchain_process/proxy.js

```diff
@@ -13,16 +13,30 @@ const utils = require('../../utils/utils');
 const WebSocket = require('ws');
 const WsParser = require('simples/lib/parsers/ws');
 
+const hex = (n) => {
+  let _n = n.toString(16);
+  return _n.length === 1 ? '0' + _n : _n;
+};
+
 const parseJsonMaybe = (string) => {
   let object;
-  try {
-    if (string && typeof string === 'string') object = JSON.parse(string);
-  } catch (e) {
-    console.error('Error parsing string as JSON', string);
-  } finally {
-    // eslint-disable-next-line no-unsafe-finally
-    return object;
+  if (typeof string === 'string') {
+    if (string) {
+      try {
+        object = JSON.parse(string);
+      } catch(e) {
+        if (Array.from(Buffer.from(string)).map(hex).join(':') !==
+            '03:ef:bf:bd') {
+          console.error(`Proxy: Error parsing string as JSON '${string}'`);
+        }
+      }
+    } else {
+      console.error('Proxy: Expected a non-empty string');
+    }
+  } else {
+    console.error(`Proxy: Expected a string but got type '${typeof string}'`);
```

Comment:
Do you think it might be a good idea to check for an object as well? Or do we never see a case when objects come through the proxy?

```
} else if (string === Object(string)) // tests for an object type (better than typeof(string) === 'object')
  object = string;
}
```

Reply:
If something other than a string comes through the proxy then it's something unexpected as all messages should be strings that can be parsed as JSON, i.e. JSON-RPC messages.